app/controllers/applicants_controller.rb

```diff
@@ -175,19 +184,17 @@ def search
 
 private
 
-  def login_applicant(applicant)
```

Comment:
This method is removed because it is only used in this controller, upon registering applicants. Since auto login upon registration is not possible anymore, this method is unnecessary.